alembic/env.py

```
@@ -0,0 +1,90 @@
+from db.base import metadata as base_metadata  # noqa: E402
+from db.engine import get_database_url  # noqa: E402
+import db.models  # noqa: E402,F401  (registers tables on metadata)
+from sqlmodel import SQLModel  # noqa: E402
+
+config = context.config
+
+if config.config_file_name is not None:
+    try:
+        fileConfig(config.config_file_name)
+    except Exception:
+        pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)